src/collections/_documentation/platforms/android/timber.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Timber
+sidebar_order: 2
+---
+
+The `sentry-android-timber` library provides [Timber](https://github.com/JakeWharton/timber) support for Sentry via [Timber Tree](https://github.com/JakeWharton/timber/blob/10f0adce3921ad2929ddf2f3b7fecda2cf3148a5/timber/src/main/java/timber/log/Timber.kt#L20) that sends events and breadcrumbs to Sentry. Once this integration is configured you can use Timber’s static API.
+
+The source can be found [on GitHub](https://github.com/getsentry/sentry-android/tree/master/sentry-android-timber/src/main/java/io/sentry/android/timber).
+
+### Installation
+
+1. In order to add the Timber integration, you have to do a [manual initialization](https://docs.sentry.io/platforms/android/#manual-initialization) of the Android SDK.
+
+2. Add the `sentry-android-timber` dependency
+
+Using Gradle:
+
+```groovy
+implementation 'io.sentry:sentry-android:{version}'
+implementation 'io.sentry:sentry-android-timber:{version}' // version >= 2.2.0
+```
+
+3. Initialize and add the `SentryTimberIntegration`
```

Comment:
@MimiDumpling  do you know how to keep a sequence instead of starting a new one?

eg this is the 3rd item, but its shown 1. in the UI, probably because there's this code snippet in between :(

Reply:
<img width="505" alt="Screenshot 2020-07-10 at 14 48 58" src="https://user-images.githubusercontent.com/5731772/87156203-99e84e00-c2bc-11ea-819d-3cecb3c8ce2e.png">

Reply:
agh. I've run into this problem, too. Haven't figured out a solution. @cameronmcefee might know.

Reply:
You need to indent your paragraphs and codesamples that belong to the bulletpoint. Right now they are at the top-level and therefore jekyll does not see them belonging to the list.

Reply:
to the point :) thanks